public/components/Tag/TagEdit.react.js

```diff
@@ -14,23 +19,37 @@ export default class TagEdit extends React.Component {
       }));
     }
 
+    //This will contain the logic for different tag forms (keyword vs contributor etc...)
```

Comment:
This will probably end up being separate components for each type before it gets unreasonable